docs/docs/10.8-pure-render-mixin.md

```diff
@@ -6,6 +6,8 @@ prev: update.html
 next: perf.html
 ---
 
+> Note: React 15.3 added the [React.PureComponent](https://github.com/facebook/react/releases/tag/v15.3.0) base class. This is a more convenient way to implement the behavior below that does not rely upon mixins. React.PureComponent is the recommend approach when working with ES2015 classes.
```

Comment:
> React.PureComponent is the recommend approach when working with ES2015 classes.

I don't like this sentence because it implies that PureComponent should always be used (which is simply untrue; using PureComponent when your component takes in children is generally counterproductive).  I'd just remove that sentence alltogether.  It's enough to say that it is convenient and doesn't rely on mixins.